parts/kuberneteskubelet.service

```diff
@@ -45,7 +45,8 @@ ExecStart=/usr/bin/docker run \
         --azure-container-registry-config=/etc/kubernetes/azure.json \
         --hairpin-mode=promiscuous-bridge \
         --network-plugin=${KUBELET_NETWORK_PLUGIN} \
-        --v=2
+        --feature-gates=Accelerators=true \
```

Comment:
I believe this is needed for GPU support. Since the templating system doesn't currently support these files (we can't put a conditional in here), we'll need another way to gate.